### PR TITLE
docs: parachain onboarding locally

### PR DESCRIPTION
**Pull request by Daanvdplas**
The following guide explains how to launch a parachain to a local test network. The guide `launching-on-paseo.md` is almost and hasn't been edited. I think this should be removed.

There is still a lot of room for improvement but not the scope of this work.

### CODE REVIEW

**Comment by Daanvdplas** [comment]
[sc-1711]

**Comment by Daanvdplas** [comment]
[sc-1778]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 1]
```diff
@@ -51,68 +35,43 @@ name = "charlie"
 validator = true
 ```
 
-> Interesting Fact: We have three validator nodes so that we can use Polkadot's Warp Sync to quickly sync with the network. The minimum requirement for Warp Sync are three validator nodes.
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:54]: Again, why remove it?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 2]
```diff
@@ -51,68 +35,43 @@ name = "charlie"
...
+```bash
+pop call parachain --pallet Configuration --function set_coretime_cores --args "1" --url ws://localhost:57731/ --suri //Alice --sudo --skip-confirm
 ```
 
-It will take some time for the parachain to build. In the meantime, we can start setting up our accounts.
+> Note: we are calling the specified rpc port `577313` which is specified in the created `network.toml` file to interact the validator `alice`.
+
+Second, assign the core to the on demand pool:
+```bash
+pop call parachain --url ws://localhost:57731 --call 0xff004a0400000a000000040100e100 --suri //Alice --skip-confirm
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:67]: Use `pop call chain`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 3]
```diff
@@ -51,68 +35,43 @@ name = "charlie"
...
-```
+As of now, Paseo local doesn't provide cores to validate parachain blocks on demand. We will have to make 2 calls to Paseo
+using `Alice` as admin account.
 
-```
-cd my-parachain
-pop build
+First, configure Paseo to set coretime cores to `1`:
+```bash
+pop call parachain --pallet Configuration --function set_coretime_cores --args "1" --url ws://localhost:57731/ --suri //Alice --sudo --skip-confirm
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:60]: Use `pop call chain`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 2 above)
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:67]: Why the call_data here? Is not clear what we are doing if we specify the extrinsic we are calling?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 4]
```diff
@@ -161,31 +124,39 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
 
 > This your key (Polkadot account). Save the secret phrase in a vault securely and never share it.
 
-Now that we have a stash account, we need to fund this account with some tokens so that it has funds to perform transactions on behalf of the collator such as transactions related to onboarding your parachain.
-
-Add the stash account to the [Polkadot Signer extension](https://polkadot.js.org/) so that you can see your account appear in the [PolkadotJs Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpaseo-rpc.dwellir.com#/accounts) UI.
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: Why remove this part?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 5]
```diff
@@ -209,32 +180,48 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
...
+◆  Generated files:
+│  ● Plain text chain specification file generated at: ./chain-spec.json
+│  ● Raw chain specification file generated at: ./chain-spec-raw.json
+│  ● WebAssembly runtime file exported at: ./para-2000.wasm
+│  ● Genesis State file exported at: ./para-2000-genesis-state
+│  
+└  Need help? Learn more at https://learn.onpop.io
 ```
-
-This will output the following:
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: Is not interesting to keep this?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 6]
```diff
@@ -417,51 +396,112 @@ curl -H "Content-Type: application/json" \
...
+│  
+◇  Do you want to submit the extrinsic?
+│  Yes 
+│
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.39.09 AM.png" alt=""><figcaption><p>ParaThread Onboarding</p></figcaption></figure>
+Event(s):
 
-Once onboarded, we now need to use sudo privileges (Alice account) on the Paseo Relay chain to "force" a lease for our parachain so that it can begin to use produce blocks.
+Registrar::Reserved: { para_id: (2000), who: ((212, 53, 147, 199, 21, 253, 211, 28, 97, 20, 26, 189, 4, 169, 159, 214, 130, 44, 133, 88, 133, 76, 205, 227, 154, 86, 132, 231, 165, 109, 162, 125)) }
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:433]: Output is outdated.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 7]
```diff
@@ -417,51 +396,112 @@ curl -H "Content-Type: application/json" \
...
-It will take a moment for the ParaThread to onboard:
+ 
+```bash
+┌   Pop CLI : Call a parachain
+│
+◇  What would you like to do?
+│  Reserve a parachain ID 
+│
+◇  Signer of the extrinsic:
+│  <PUBLIC KEY STASH ACCOUNT>
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:425]: That's wrong no?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 8]
```diff
@@ -51,68 +35,43 @@ name = "charlie"
...
-```
-cd my-parachain
-pop build
+First, configure Paseo to set coretime cores to `1`:
+```bash
+pop call parachain --pallet Configuration --function set_coretime_cores --args "1" --url ws://localhost:57731/ --suri //Alice --sudo --skip-confirm
 ```
 
-It will take some time for the parachain to build. In the meantime, we can start setting up our accounts.
+> Note: we are calling the specified rpc port `577313` which is specified in the created `network.toml` file to interact the validator `alice`.
```
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:63]:
```suggestion
> Note: we are calling the specified rpc port `57731` which is specified in the created `network.toml` file to interact the validator `alice`
```

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 9]
```diff
@@ -4,42 +4,26 @@ description: >-
   Testnet Locally
 ---
 
-# Launching your parachain manually
+# Launch your parachain on a Local Test Network.
 
-## Introduction
```
[the remark this answers is not in the document]
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: You feel it is necessary?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 1 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:54]: You feel it is necessary? Someone wants to deploy a parachain, there is already a lot of info.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 4 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: Do we need it? We do everything with pop cli

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 5 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: See the bash script saying exactly the same

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 7 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:427]: Why?

Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:427]: Here is the Secret key URI, not the public. maybe just leave it as `<STASH ACCOUNT>`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 5 above)
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: You are right, sorry

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 4 above)
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: True!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: I don't see it as a bad content a brief explanation on what Paseo is, maybe is too long. But not blocker for me.

[the remark this answers is not in the document]
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: Moved it back

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 6 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Updated

**Review by AlexD10S — APPROVED** [review]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 10]
```diff
@@ -244,18 +247,18 @@ Make sure to edit your chain spec and:&#x20;
 
 * **add your account and session keys**
 * **specify the starting balance of specific accounts**
-* **add the account that will be the sudo account for your parachain**
+* **add the account that will be the sudo account for your chain**
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:250]: I am a bit skeptical about inviting users to modify the plain spec directly instead of having code in the project that gives them the expected genesis.

But I think this is a different conversation than what this PR involves.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 11]
```diff
@@ -417,51 +412,117 @@ curl -H "Content-Type: application/json" \
...
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.46.09 AM.png" alt=""><figcaption><p>Force a lease</p></figcaption></figure>
+ 
+```bash
+┌   Pop CLI : Call a chain
+│
+◇  What would you like to do?
+│  Reserve a parachain ID 
+│
+◇  Signer of the extrinsic:
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:440]: We could include `--use-wallet` to exemplify a more solid flow.
Otherwise we can come back once the feature its merged!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 12]
```diff
@@ -417,51 +412,117 @@ curl -H "Content-Type: application/json" \
...
+◇  Enter the value for the parameter: id
+│  2000
+│
+◇  The value for `genesis_head` might be too large to enter. You may enter the path to a file instead.
+│  para-2000-genesis-state
+│
+◇  The value for `validation_code` might be too large to enter. You may enter the path to a file instead.
+│  para-2000.wasm
+│
+◇  Signer of the extrinsic:
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:476]: Same comment than above regarding `--use-wallet`.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 13]
```diff
@@ -417,51 +412,117 @@ curl -H "Content-Type: application/json" \
...
+```
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.50.18 AM.png" alt=""><figcaption><p>Parachain Tab</p></figcaption></figure>
+Your chain is now registered on Paseo!
 
-After a few moments, you should see the parachain producing blocks. The included / backed will update with the Relay chain's block number in which your parachain's block was included and backed. You can also check your collator logs to confirm.
+### Buy an On Demand Core
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 2.21.37 PM.png" alt=""><figcaption><p>Notice the included / backed</p></figcaption></figure>
+Now we need to buy a core to have Paseo validate a block.
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:499]: Not 100% correct. One only buy the finalization of one core.
More of a pay finalization as you produce blocks.

Buying a core would be something more akin to buying bulk coretime. Which in reality isn't cores either what is being sold, but regions. :)

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 14]
```diff
@@ -417,51 +412,117 @@ curl -H "Content-Type: application/json" \
...
+         para_id: Id(2000)
+         manager: <STASH ACCOUNT>
+...
+```
 
-<figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 11.50.18 AM.png" alt=""><figcaption><p>Parachain Tab</p></figcaption></figure>
+Your chain is now registered on Paseo!
 
-After a few moments, you should see the parachain producing blocks. The included / backed will update with the Relay chain's block number in which your parachain's block was included and backed. You can also check your collator logs to confirm.
+### Buy an On Demand Core
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:497]: Might be interesting pointing to the wiki - https://wiki.polkadot.network/docs/learn-agile-coretime
As a reference in case someone arrives to this point and is not aware of what on demand coretime is. :)

**Review by al3mart — CHANGES_REQUESTED** [review]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 15]
```diff
@@ -4,18 +4,18 @@ description: >-
...
+A typical development workflow for launching your chain on Polkadot:&#x20;
 
-* Run your parachain **locally** using Pop CLI.&#x20;
-  * Under the hood, Pop CLI launches your parachain to Paseo Local Testnet automatically for development purposes:
+* Run your chain **locally** using Pop CLI.&#x20;
+  * Under the hood, Pop CLI launches your chain to Paseo Local Testnet automatically for development purposes:
     * [Run your parachain on Paseo](running-your-parachain.md)
-* When ready to test your parachain in a live environment with other parachains:
+* When ready to test your chain in a live environment with other chains:
   * Use this guide to mimic the manual onboarding process for Paseo Testnet locally.&#x20;
```
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]:
```suggestion
  * Use this guide to mimic the manual onboarding process for a Paseo Local Testnet.&#x20;
```

As we call a production Paseo network `Paseo Live Testnet`, I suggest following the format `Paseo [Live | Local] Testnet` to keep the consistency.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 16]
```diff
@@ -4,18 +4,18 @@ description: >-
...
   * When comfortable with manually onboarding locally, then use the next guide to onboard to Paseo Live Testnet.
     * [Launching your parachain on Paseo Testnet](launching-your-parachain-on-polkadot/launching-on-paseo.md)
```
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:21]: And then here it is called `Paseo Testnet`, a bit confusing to me whether this is Live Testnet or Local Testnet.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 15 above)
Comment (chungquantin) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: I guess this is Paseo Live Testnet?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 17]
```diff
@@ -4,18 +4,18 @@ description: >-
...
 
 [Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet.
 
-A typical development workflow for launching your parachain on Polkadot:&#x20;
+A typical development workflow for launching your chain on Polkadot:&#x20;
 
-* Run your parachain **locally** using Pop CLI.&#x20;
-  * Under the hood, Pop CLI launches your parachain to Paseo Local Testnet automatically for development purposes:
+* Run your chain **locally** using Pop CLI.&#x20;
+  * Under the hood, Pop CLI launches your chain to Paseo Local Testnet automatically for development purposes:
```
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:16]: Here it is used `Paseo Local Testnet`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 18]
```diff
@@ -26,20 +26,22 @@ A typical development workflow for launching your parachain on Polkadot:&#x20;
 
 Let's get started.
 
-## Spin up Paseo locally
+## Launch Paseo Local
```
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:29]:
```suggestion
## Launch Paseo Testnet on Local
```

or 

```suggestion
## Launch Paseo Local Testnet
```

**Review by chungquantin — CHANGES_REQUESTED** [review]
We should choose a consistent naming convention for "Paseo Testnet" throughout the document. For clarity and readability, I suggest using `Paseo [Live | Local] Testnet` as the standard term in all instances.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 19]
```diff
@@ -51,68 +53,43 @@ name = "charlie"
...
+pop call chain --url ws://localhost:57731 --call 0xff004a0400000a000000040100e100 --suri //Alice --skip-confirm
+```
 
 ## Setting Up Accounts
 
-First, we will need to set up a stash account to do transactions on Paseo on behalf of our collator.
+First, we will need to set up a stash account to do transactions on Paseo on behalf of our collator that we will launch later in this guide.
 
-> A collator is the parachain node that will be running your parachain.
+> A collator is the parachain node that will be running for your parachain.
```
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:92]:
```suggestion
> A [collator](https://wiki.polkadot.network/docs/learn-collator) is the parachain node that will be running for your parachain.
```

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 10 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:250]: Yes out of scope I'm afraid

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 13 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:499]: validated and finalised*

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 14 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:497]: Nice one!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 15 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: Yes, great suggestion

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 19 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:92]: Great suggestion, thank you!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 15 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: I am calling it Paseo Local and Paseo, the latter being live. Also removing the word testnet as that is explained via a link at the top.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 20]
```diff
@@ -209,32 +194,50 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
...
 
-* Your parachain's chain specification file
-  * The plain text and the raw version e.g. `chain-spec.json` and `chain-spec-raw.json`
-* Your parachain's initial genesis state e.g. `para-2000-genesis-state`
-* Your parachain's Wasm runtime e.g. `para-2000.wasm`
+```
+cd my-chain
+pop build spec --profile release --id 2000 --type local --relay paseo-local --protocol-id my_chain --chain local --genesis-state --genesis-code
+```
+```bash
```
Comment (evilrobot-01) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:220]: Do we still need this output if we have the gif?

Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:214]: For copy pasting it can still be handy perhaps.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 14 above)
Comment (evilrobot-01) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:491]: 'Buy a core' doesnt sound right to me - we are acquiring coretime for a single block where this sounds like you have bought it outright and own it, which is not true.

https://wiki.polkadot.network/docs/learn-agile-coretime#on-demand-coretime

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 21]
```diff
@@ -320,65 +317,58 @@ It should look similar to the below:
...
 > path/to/polkadot-sdk/target/debug/substrate-node key generate-node-key --file=secret_ed25519 --chain=./chain-spec-raw.json
 > ```
 >
-> <pre><code><strong>mv secret_ed25519 data/chains/my_parachain/network
+> <pre><code><strong>mv secret_ed25519 data/chains/my_chain/network
 > </strong></code></pre>
->
->
+
+### Run Collator
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:371]: Note that the above node key is placed in the `.data/` directory assuming that the user will run the collator copy/pasting the instructions.

More concretely the node key generated in the previous step will only be used if `--base-path ./data ` is provided.

Authorities need to be run with a specific node key. Otherwise `--unsafe-force-node-key-generation` can be provided to delegate the key generation to the node itself (this presents other nuances). But this is OK for local environments.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md [hunk 22]
```diff
@@ -1,45 +1,41 @@
...
 ---
 
-# Launching your parachain manually
+# Onboard a Chain to Paseo Local
 
-## Introduction
+[Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet. Paseo Local is for running Paseo on a local machine.
 
-[Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet.
+A typical development workflow for launching a chain on Polkadot:&#x20;
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:10]: Afraid of adding content to this already long doc. 
But I feel we could use some extra pointers for the bullet points bellow.

#1. Is very useful when testing the runtime of your chain. You are not really worried about how the onboarding process works. But you need to see the runtime executing some blocks. "live" integration tests with other chains and such.

#2. Is only helpful when you are trying to learn how to onboard your chain to an already running relay chain (like Westend, Paseo, Kusama or Polkadot). It is very helpful to nail down the concepts around bootstrapping collators, seeing the process and taking a few leanings from it.
This document.

The learnings in #2. are mostly equivalent to #3. in terms of what the user needs to know.
I'd argue that we can port most of the new content from here into 3. (or vice versa) and get rid of all the details around configuring  the relay chain. It is too nuanced, and possibly it changes in future versions of the SDK.

Ideally no user would need to know about the scheduling gymnastics the relay does.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 20 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:214]: You are right, removed the command and output.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 22 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: This was a bit hard to follow what you meant. Please see the latest changes and shout if there is anything pressing missing (or create an issue if not pressing).

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 21 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Added a little `Note`

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md [hunk 23]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (Daanvdplas) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:1]: Didn't change much in this guide, just copy pasted.

File: pop-cli-for-appchains/guides/spinning-up-a-polkadot-network/spinning-up-the-polkadot-network.md [hunk 24]
```diff
@@ -4,53 +4,73 @@ description: How to spin up the Paseo Relay chain locally
 
 # Spinning up Paseo
 
-Use Pop CLI to spin up Paseo locally!
+Lets create a configuration file to launch Paseo Local:
```
Comment (Daanvdplas) [inline pop-cli-for-appchains/guides/spinning-up-a-polkadot-network/spinning-up-the-polkadot-network.md:7]: Added configuration to this file. As we would like to create a command `pop up paseo` this guide will become quickly irrelevant.

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md [hunk 25]
```diff
@@ -0,0 +1,51 @@
...
+description: The following guide shows how to acquire coretime.
+---
+
+# Coretime
+
+In order to get a chain's block validated and finalised by the Relay chain it needs to acquire a [core](https://wiki.polkadot.network/docs/learn-agile-coretime).
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:7]:
```suggestion
In order to get a chain's block validated and finalised by the Relay chain it needs to acquire [coretime](https://wiki.polkadot.network/docs/learn-agile-coretime).
```

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md [hunk 26]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (al3mart) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:8]: "pop allows users to obtain on demand coretime as easily as executing one command:"

**Review by al3mart — APPROVED** [review]
This is way more digestible! :+1: 

Left a couple of comments, but I'll leave it up to you to include, @Daanvdplas !
Good job.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md (hunk 14 above)
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Changed to acquire a core

**Review by al3mart — APPROVED** [review]
